.github/workflows/build.yml

```diff
@@ -162,9 +169,10 @@ jobs:
           role-to-assume: ${{ contains(fromJSON('["main","subset"]'), github.ref_name) && secrets.AWS_KMS_PROD_ROLE_ARN || secrets.AWS_KMS_STAGE_ROLE_ARN }}
           # TODO figure out if there's some way we could make our secrets ternaries here more DRY without major headaches 🙈
       - name: Sign
+        id: sign
         if: env.shouldSign == 'true'
         env:
-          AWS_KMS_REGION:  ${{ contains(fromJSON('["main","subset"]'), github.ref_name) && secrets.AWS_KMS_PROD_REGION  || secrets.AWS_KMS_STAGE_REGION }}
+          AWS_KMS_REGION:  ${{ job.build.steps.auth.with.args.aws-region }}
```

Comment:
Just to make sure -- GitHub _does_ still hide references to this secret even when it's referenced indirectly like this, right?

(I know with Jenkins the secret has to be scoped inside the block that provides access to it for it to be hidden, but I haven't used GitHub's secrets enough to know the details of how they work / where and how they're hidden.)

Reply:
It will mask all output that matches a secret anywhere I believe. It will even mask base64 encoded output of a secret as well. Although, it won't mask `echo $SECRET | rev` last time I checked 🙈